dask/highlevelgraph.py

```diff
@@ -394,7 +395,7 @@ def __dask_distributed_pack__(
 
         # Calculate dependencies without re-calculating already known dependencies
         # - Start with known dependencies
-        dependencies = known_key_dependencies.copy()
```

Comment:
Probably better to just type the input as `dict`. But not worth spending a bunch of time here

Reply:
cool -- going to leave this as-is for now